.github/workflows/test-ruby.yml

```diff
@@ -148,6 +148,93 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
+  test-libvips:
```

Comment:
We should probably use a re-usable workflow for this to deduplicate the code, but this can be done in a further PR